middle_end/flambda/to_cmm/un_cps.ml

```diff
@@ -1321,7 +1364,9 @@ and let_dynamic_set_of_closures env res body closure_vars decls elts =
     List.fold_left2 (fun acc cid v ->
       let v = Var_in_binding_pos.var v in
       let e, effs = get_closure_by_offset env soc_cmm_var cid in
-      let_expr_bind body acc v e effs
+      (* It doesn't seem important to know the number of occurrences here. *)
```

Comment:
Actually, it's one of the few places where it's reasonable to actually take into account the number of occurrences, in order to avoid creating unnecessary bindings (I'm not sure if such bindings could be eliminated by later passes). It's not crucial, but it would be nice to not bind closures that are not used at all (e.g. in cases where only one of the recursive functions is returned).

Reply:
This has been fixed